### PR TITLE
Add additional status codes to eeLink protocol

### PR DESCRIPTION
**Pull request by casswarry0**
Decode all Status bits 0...11 to get additional information about: device in motion, device charging, status of GPS module, etc.

### CODE REVIEW

**Comment by tananaev** [comment]
Please attach the protocol documentation.

**Comment by casswarry0** [comment]
[eeLink Device Protocol V2.0.3 (A4).pdf](https://github.com/traccar/traccar/files/10451102/eeLink.Device.Protocol.V2.0.3.A4.pdf)
[eeLink Device Protocol V2.2 _A4_.pdf](https://github.com/traccar/traccar/files/10451105/eeLink.Device.Protocol.V2.2._A4_.pdf)

File: src/main/java/org/traccar/protocol/EelinkProtocolDecoder.java [hunk 1]
```diff
@@ -270,9 +270,20 @@ private Position decodeNew(DeviceSession deviceSession, ByteBuf buf, int type, i
...
+            if (BitUtil.check(status, 3)) { // accelerometer supported
+                position.set(Position.KEY_ARMED, BitUtil.check(status, 4));
+                position.set(Position.KEY_MOTION, BitUtil.check(status, 9));
+            }
+            if (BitUtil.check(status, 5)) { // relay control supported
+                position.set(Position.KEY_BLOCKED, BitUtil.check(status, 6));
+            }
+            if (BitUtil.check(status, 7)) { // external charging supported
+                position.set(Position.KEY_CHARGE, BitUtil.check(status, 8));
+            }
```
Comment (tananaev) [inline src/main/java/org/traccar/protocol/EelinkProtocolDecoder.java:285]: Please remove the comments.

Comment (casswarry0) [reply src/main/java/org/traccar/protocol/EelinkProtocolDecoder.java]: Comments are removed.

**Comment by tananaev** [comment]
Merged, thank you.

**Pull request merged**